_sass/_layouts--global.scss

```diff
@@ -11,3 +11,12 @@ body {
     flex: 1;
   }
 }
+
+body {
+  // left the gap for header navbar
+  padding-top: $size--header-height;
```

Comment:
hmm it does, but actually on some pages its over the content on top, but its ok on those pages we can just add the class I mention to the body